modules/media/php/NDB_Menu_Filter_media.class.inc

```diff
@@ -129,6 +130,10 @@ class NDB_Menu_Filter_Media extends NDB_Menu_Filter
 
         $this->query = $query;
 
+        if (!$user->hasPermission('access_all_profiles')) {
```

Comment:
@driusan 

Is there a reason why this change is not on the 17.1-dev branch? is it because the 17.1 branch is not based off of the released v17.0.0 branch?